src/styles/card.scss

```
@@ -45,14 +59,15 @@
     text-transform: uppercase;
     color: #fff;
 
-    background-color: #00acdc;
-    border: 1px solid #00acdc;
+    background-color: $color;
+    border: 1px solid $color;
     border-radius: 5px;
-  }
-}
 
-.card__btn:hover {
-  color: #00acdc;
+    transition: background-color 1s, color 1s, transform 1s;
```

```suggestion
    transition: background-color 1s, color 1s;
```
Здесь `transform` лишний, мы кнопку никак не трансформируем при `:hover`, только цвета меняем.